src/node/eth.rs

```diff
@@ -2226,7 +2226,7 @@ mod tests {
             .await
             .expect("failed getting filter changes")
         {
-            FilterChanges::Logs(result) => assert_eq!(2, result.len()),
+            FilterChanges::Logs(result) => assert_eq!(3, result.len()),
```

Comment:
Hmmm, why did this increase? I'm worried that we were even expected 2 logs here 🤔

Reply:
I'll defer and have a follow-up investigation

Reply:
Yes, it is odd - now the logs are
```
[Log { address: 0x000000000000000000000000000000000000800a, topics: [0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef, 0x0000000000000000000000009555be19b8e5f4c3af1ada6ec94bbc1b901b8235, 0x0000000000000000000000000000000000000000000000000000000000008001], data: Bytes("0x0000000000000000000000000000000000000000000000000000b5e620f48000"), block_hash: Some(0xd97ba6a5ab0f2d7fbfc697251321cce20bff3da2b0ddaf12c80f80f0ab270b15), block_number: Some(1), l1_batch_number: Some(1), transaction_hash: Some(0x0101010101010101010101010101010101010101010101010101010101010101), transaction_index: Some(0), log_index: Some(0), transaction_log_index: Some(0), log_type: None, removed: Some(false) }, Log { address: 0x000000000000000000000000000000000000800a, topics: [0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef, 0x0000000000000000000000000000000000000000000000000000000000008001, 0x0000000000000000000000009555be19b8e5f4c3af1ada6ec94bbc1b901b8235], data: Bytes("0x00000000000000000000000000000000000000000000000000005af3107a4000"), block_hash: Some(0xd97ba6a5ab0f2d7fbfc697251321cce20bff3da2b0ddaf12c80f80f0ab270b15), block_number: Some(1), l1_batch_number: Some(1), transaction_hash: Some(0x0101010101010101010101010101010101010101010101010101010101010101), transaction_index: Some(0), log_index: Some(1), transaction_log_index: Some(1), log_type: None, removed: Some(false) }, Log { address: 0x000000000000000000000000000000000000800a, topics: [0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef, 0x0000000000000000000000000000000000000000000000000000000000008001, 0x0000000000000000000000009555be19b8e5f4c3af1ada6ec94bbc1b901b8235], data: Bytes("0x000000000000000000000000000000000000000000000000000007f544a44c00"), block_hash: Some(0xd97ba6a5ab0f2d7fbfc697251321cce20bff3da2b0ddaf12c80f80f0ab270b15), block_number: Some(1), l1_batch_number: Some(1), transaction_hash: Some(0x0101010101010101010101010101010101010101010101010101010101010101), transaction_index: Some(0), log_index: Some(2), transaction_log_index: Some(2), log_type: None, removed: Some(false) }]
```
while previously they were just
```
[Log { address: 0x000000000000000000000000000000000000800a, topics: [0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef, 0x000000000000000000000000d0cf56cb2ae8a8a0e61ffa65b0b92262eb3ac39a, 0x0000000000000000000000000000000000000000000000000000000000008001], data: Bytes("0x00000000000000000000000000000000000000000000000000005af3107a4000"), block_hash: Some(0xd97ba6a5ab0f2d7fbfc697251321cce20bff3da2b0ddaf12c80f80f0ab270b15), block_number: Some(1), l1_batch_number: Some(1), transaction_hash: Some(0x0101010101010101010101010101010101010101010101010101010101010101), transaction_index: Some(0), log_index: Some(0), transaction_log_index: Some(0), log_type: None, removed: Some(false) }, Log { address: 0x000000000000000000000000000000000000800a, topics: [0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef, 0x0000000000000000000000000000000000000000000000000000000000008001, 0x000000000000000000000000d0cf56cb2ae8a8a0e61ffa65b0b92262eb3ac39a], data: Bytes("0x0000000000000000000000000000000000000000000000000000000000000000"), block_hash: Some(0xd97ba6a5ab0f2d7fbfc697251321cce20bff3da2b0ddaf12c80f80f0ab270b15), block_number: Some(1), l1_batch_number: Some(1), transaction_hash: Some(0x0101010101010101010101010101010101010101010101010101010101010101), transaction_index: Some(0), log_index: Some(1), transaction_log_index: Some(1), log_type: None, removed: Some(false) }]
```
I see nothing wrong with either, but why they should change depending on fees I have no idea...